.gitattributes

```diff
@@ -1 +1,3 @@
 conda_pack/_version.py export-subst
+# GitHub syntax highlighting
+pixi.lock linguist-language=YAML linguist-generated=true
```

Comment:
Was that intentional?

Reply:
Not really. I manage the development locally with `pixi` but we're not using it officially in the repo, so we don't need this set.